src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +98,53 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={searchedQuery}
+              onChange={(event) => setQuery(event.target.value)}
             />
           </div>
-
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isError ? (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          ) : ''}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button is-light',
+                { 'is-loading': isSubmitted },
+              )}
+              disabled={!searchedQuery}
             >
               Find a movie
```

Comment:
```suggestion
              {!movie
                  ? 'Find a movie'
                  : 'Search again'}
```
first screen from your app and second from example

![image](https://user-images.githubusercontent.com/87998640/236564791-b8a31807-03c3-4c41-88a1-1c7bb8e8bf94.png)
![image](https://user-images.githubusercontent.com/87998640/236564871-310dc136-691c-4fba-aca8-f5928a7b6f48.png)